unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HomePoint/IHomeLocationController.cs

```diff
@@ -0,0 +1,7 @@
+using UnityEngine;
+
+public interface IHomeLocationController
+{
+    void SetHomeScene(string location);
```

Comment:
I think the interface should not support this overload, because it could lead to bugs since a string could have a wrong format or anything you want. I think we should only support `Vector2` as a parameter to get a more controlled usage of the feature.

Reply:
alright, will change in a bit :)